xapian-core/tests/api_backend.cc

```diff
@@ -1401,3 +1401,35 @@ DEFINE_TESTCASE(embedded1, singlefile) {
 
     return true;
 }
+
+static string get_db_table_path(const string& db_path){
```

Comment:
You not using the Xapian coding style consistently - especially with regards to whitespace which is missing in a lot of places.

There's a script to help with this, which catches the more common issues:

```
git diff origin/master | xapian-maintainer-tools/xapian-check-patch
```